yt/fields/field_info_container.py

```diff
@@ -392,6 +392,7 @@ def create_function(f):
             self.alias(name, tuple_name)
         else:
             self[name] = DerivedField(name, sampling_type, function, **kwargs)
+        return None
```

Comment:
I see `return None` statement all over the place ; isn't it equivalent to just return None? Why is it better (if it is)?

Reply:
it *is* equivalent to `return`. In the context of functions that may return anything else, it is preferable to make it explicit that some branch in fact return None. It is "better" in the sense that
> explicit is better than implicit
(Zen of Python)

I'm convinced it will come in handy for YTEP 0038 too :)